src/dialects/ibmi/index.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+
+const _ = require('lodash');
+const { AbstractDialect } = require('../abstract');
+const ConnectionManager = require('./connection-manager');
+const Query = require('./query');
+const { QueryGenerator } = require('./query-generator');
+const { IBMiQueryInterface } = require('./query-interface');
+const DataTypes = require('../../data-types').ibmi;
+
+class IBMiDialect extends AbstractDialect {
+
+  static supports = _.merge(
+    _.cloneDeep(AbstractDialect.supports),
+    {
+      DEFAULT: true,
```

Comment:
no need to override an option if it's already the same in AbstractDialect (we actually went ahead and cleaned up the other dialects)